Create rule S6505: Allowing shell scripts execution during package installation is security-sensitive (APPSEC-483) (#1584)
SonarSource/rspec: rules/S6505/ask-yourself.adoc

== Ask Yourself Whether

> * At least one of the application's dependencies requires the execution of install scripts to function correctly.

Reviewer: A suggestion to make it a bit clear:
```suggestion
* The execution of dependency installation scripts is required for the application to function correctly.
```
